launcher/FileIgnoreProxy.cpp

```diff
@@ -254,3 +255,26 @@ bool FileIgnoreProxy::filterAcceptsColumn(int source_column, const QModelIndex&
 
     return true;
 }
+
+bool FileIgnoreProxy::filterAcceptsRow(int sourceRow, const QModelIndex& sourceParent) const
+{
+    QModelIndex index = sourceModel()->index(sourceRow, 0, sourceParent);
+    QFileSystemModel* fsm = qobject_cast<QFileSystemModel*>(sourceModel());
+
+    auto fileInfo = fsm->fileInfo(index);
+    return !ignoreFile(fileInfo);
+}
+
+bool FileIgnoreProxy::ignoreFile(QFileInfo fileInfo) const
+{
+    auto fileName = fileInfo.fileName();
+    auto path = relPath(fileInfo.absoluteFilePath());
+    return (path.startsWith("..") ||  // just in case ignore files outside the gameroot
```

Comment:
this shouldn't be needed
if this is happening, surely there'd be other issues!

Reply:
Yeah it works without it